src/euroeval/dataset_configs/finnish.py

```diff
@@ -0,0 +1,59 @@
+"""All Finnish dataset configurations used in EuroEval."""
+
+from ..data_models import DatasetConfig
+from ..languages import FI
+from ..tasks import COMMON_SENSE, LA, NER, RC, SENT, SUMM
+
+### Official datasets ###
+
+SCANDISENT_FI_CONFIG = DatasetConfig(
+    name="scandisent-fi",
+    pretty_name="the truncated version of the Finnish part of the binary sentiment "
+    "classification dataset ScandiSent",
+    huggingface_id="EuroEval/scandisent-fi-mini",
+    task=SENT,
+    languages=[FI],
```

Comment:
This dataset only has the 'positive' and 'negative' labels, so we need to overwrite the default labels here:

```suggestion
    languages=[FI],
    _labels=["negative", "positive"],
```